src/main/java/uk/gov/hmcts/darts/audio/config/AudioConfigurationProperties.java

```diff
@@ -34,5 +34,7 @@ public class AudioConfigurationProperties {
     private List<String> allowedMediaFormats = new ArrayList<>();
     private Integer maxFileSize;
     private Duration allowableAudioGapDuration;
```

Comment:
@garethmorriscgi This is an example of a `Duration` field in the config, like what @johnrace0 was mentioning earlier